DuckDuckGo/AppIcon.swift

```diff
@@ -27,6 +27,11 @@ enum AppIcon: String, CaseIterable {
     case purple
     case black
 
+    /// Returns a user facing string representation of the app icon.
+    var name: String {
```

Comment:
Added this "shadow" of `rawValue` to make it more explicit that this is a user facing representation. It could be considered to be changed to something like `"\(rawValue) app icon"` in the future, as the color itself doesn't necessarily fully describe the element; although I left this out for now as the context should be enough.